web/client/components/data/download/FeatureEditorButton.jsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-import Button from '../featuregrid/toolbars/TButton';
+import withHint from "../featuregrid/enhancers/withHint";
+import TButtonComp from "../featuregrid/toolbars/TButton";
+const TButton = withHint(TButtonComp);
```

Comment:
why this change?

Reply:
i guess it has introduced this
![image](https://user-images.githubusercontent.com/11991428/161998160-6aee8603-7f72-4106-afd5-22609b896d3d.png)

Reply:
Yes, tooltip was missing, but I'm wondering why you don't have this string locally? 
![image](https://user-images.githubusercontent.com/4226620/162071960-1cd50daf-6ada-449f-ac57-fa5d8ae41512.png)